poller/poller.go

```diff
@@ -9,7 +9,14 @@ import (
 	"github.com/splitio/go-client/splitio/service/dtos"
 )
 
-// Poller contains cache data, splitio, and required info to interact with aplitio api
+// Fetcher is an interface contains GetCache, Start and Stop functions
+type Fetcher interface {
+	Start()
+	Stop()
+	GetCache() Cache
```

Comment:
`Cache` doesn't describe this data very well.  Is there a more descriptive name?  `SplitData`?

Reply:
Not sure if `SplitAndSegment` will be too long? or if `segment` is included in `split`'s definition then `SplitData` seems great